tiledb/dataframe_.py

```diff
@@ -82,7 +82,14 @@ class ColumnInfo:
     def __init__(self, arr_or_dtype):
         import pandas as pd
 
-        dtype = pd.core.dtypes.common.get_dtype(arr_or_dtype)
+        # compatibility for pandas < 1.2
```

Comment:
@ihnorton I removed `get_dtype` in https://github.com/TileDB-Inc/TileDB-Py/pull/516/commits/eb8af19c266e0b761d7965dd9a3cb489501a05f0

Reply:
Ok, thanks